lib/ddtrace/sync_writer.rb

```diff
@@ -41,6 +41,12 @@ def write(trace, services = nil)
       Logger.log.debug(e)
     end
 
+    # Added for interface completeness
+    def stop
+      # No cleanup to do for the SyncWriter
```

Comment:
should we do an explicit flush here?

Reply:
nvm, we always flush as soon as anything is written.. duh... ignore

Reply:
We don't have too as traces are never stored inside the `SyncWriter` and the transport is always fully executed (including I/O flushing) on each trace.